### PR TITLE
 Create integration test for note creation 

### PR DESCRIPTION
**Pull request by edwardpark**
i'll clean up the commit and force it over for a new pr but wanted to see for additional feedback for just happy path test to get this setup?

### CODE REVIEW

**Comment by colinxfleming** [comment]
Hey Edward, glad to have you! Thanks very much for your pr! 

I'll review over the weekend and offer feedback/suggestions.

File: test/integration/note_test.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        click_link 'Notes'
+        
+        within('#notes')do
+            assert_text 'Notes' #confirm notes header is visible 
+            assert has_button? 'Submit'
+        end
+    end 
+    
+    it 'should let you add a new pregnancy note' do
+        fill_in 'note_body', with: 'Sample Note Body'
```
Comment (colinxfleming) [inline test/integration/note_test.rb]: For better attribute clarity we've renamed this `full_text` -- just a heads up!

File: test/integration/note_test.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        end 
+        assert_text 'Sample Note Body'
+    end 
+
+    it 'should let you edit a pregnancy note' do
+        click_on 'edit'
+        fill_in 'note_body', with: 'Sample Note Body Edit'
+        click_on 'Submit'
+        
+        assert_text 'Sample Note Body Edit'
```
Comment (colinxfleming) [inline test/integration/note_test.rb]: I'd consider reloading the page as well to ensure that it stays persistent between pageloads.

File: test/integration/note_test.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    @pregnancy = create :pregnancy, appointment_date: nil, patient: @patient
+    visit edit_pregnancy_path(@pregnancy)
+  end
+ 
+  describe 'add patient pregnancy notes' do
+    it 'should display a pregnancy notes form and current notes' do
+        click_link 'Notes'
+        
+        within('#notes')do
+            assert_text 'Notes' #confirm notes header is visible
```
Comment (colinxfleming) [inline test/integration/note_test.rb]: I expect that the exact attributes we're looking for will get remixed a little when the page gets set up but I think your instinct here is right on!

File: test/integration/note_test.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+class PregnancyNoteTest < ActionDispatch::IntegrationTest
```
Comment (colinxfleming) [inline test/integration/note_test.rb]: Any chance you could do me a solid and give this a slightly more descriptive filename, and match the class name? Something like `note_creation_test.rb` so we can tell exactly what actions the integration test is looking at, just by the filename.

**Comment by colinxfleming** [comment]
@edwardpark looks good! Obv we don't have this quite set up yet (the frontend side of this will be our major focus on Wednesday at the meeting) but I'm super excited that you've done us the solid of kickstarting it! 

Had a few suggestions that you can tackle at your leisure or on Wednesday. 

Also, make sure you say hi if you're there on Wednesday! Not sure I've had the pleasure of meeting you yet!

**Comment by edwardpark** [comment]
@colinxfleming thanks! i'll setup and update these changes , I won't be able to actually make it this Wednesday ( out of town for a wedding ) but i'll be sure to stop by the next one after though.

**Comment by colinxfleming** [comment]
Sounds good! I expect that by then we'll have the form implemented so we may have a couple more edits, but they'll probably be pretty straightforward and trivial. 

Looking forward to meeting you! Thanks again for your work!

**Comment by colinxfleming** [comment]
(Note to self: Don't merge yet, but fixes #193)

**Pull request closed without merging**